content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst

```diff
@@ -0,0 +1,112 @@
+============
+Stock report
+============
+
+Use the stock report in Odoo *Inventory* for a detailed list of all stored products, including those
+reserved, purchased and in transit, as well as those delivered to customers.
+
+.. note::
+   The reporting feature only accessible to users with :doc:`admin access
+   <../../../../general/users/access_rights>`.
+
+To access the stock report, go to :menuselection:`Inventory app --> Reporting --> Stock`.
+
+.. image:: stock/stock-report.png
+   :align: center
+   :alt: Show the stock report, accessible by going to Inventory > Reporting > Stock.
+
+Navigate the stock report
+=========================
+
+On the stock report, the left sidebar includes :guilabel:`Warehouses` to filter products by specific
+warehouses and :guilabel:`Category`, to show products within a selected product category.
+
+In the report itself, the columns represent:
+
+- :guilabel:`Product`: name of the product.
+- :guilabel:`Unit Cost`: average inventory valuation per unit, adjusted based on purchasing costs.
```

Comment:
would a link to the documentation on valuation help? I know there is aFIFO, LIFO, AVCO, etc- and it cna affect the cost? Is that relevant to mention or link to?

Reply:
Added the costing methods doc! thank you for the suggestion :smile: